modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/info_panel_file_entry.jsp

```diff
@@ -289,14 +289,15 @@ long assetClassPK = DLAssetHelperUtil.getAssetClassPK(fileEntry, fileVersion);
 					<c:if test="<%= portletDisplay.isWebDAVEnabled() && fileEntry.isSupportsSocial() && latestVersion %>">
 
 						<%
+						String webDavHelpMessage = LanguageUtil.format(request, "webdav-help", "https://help.liferay.com/hc/en-us/articles/360028720352-Desktop-Access-to-Documents-and-Media", false);
 						String webDavURLInputId = liferayPortletResponse.getNamespace() + "webDavURLInput";
 						%>
 
 						<div class="form-group">
 							<label for="<%= webDavURLInputId %>">
 								<liferay-ui:message key='<%= TextFormatter.format("webDavURL", TextFormatter.K) %>' />
 
-								<liferay-ui:icon-help message='<%= LanguageUtil.get(request, "webdav-help") %>' />
+								<liferay-ui:icon-help message='<%= webDavHelpMessage %>' />
```

Comment:
I'm not able to click the link.

Can you?

https://user-images.githubusercontent.com/47524751/182390940-ecf5fdaa-468c-495c-8c75-41b809f97da7.mp4

Reply:
Hi @AliciaGarciaGarcia ,

You are right, the tooltip does not provide a clickable link.

From my understanding, this is due to the usage of the **liferay-ui:icon-help** taglib; it looks like the tooltip is expected to only provide text.

But, the link does work from the Access from Desktop menu.

Reply:
if the link of the icon do not work, can you delete the "read more" of the tooltip? (or talk with FI to use a better component) 
It's confusing and do not gives information to the user.

Thanks!

Reply:
Hi @AliciaGarciaGarcia ,

That's a good point.

**Regarding:**
> if the link of the icon do not work, can you delete the "read more" of the tooltip?

Sure, should I create another language key: **webdav-help-tooltip**, which does not include the 'Read more' in the message?

Reply:
Hi @AliciaGarciaGarcia ,

I have sent a new PR with updated changes: https://github.com/liferay-lima/liferay-portal/pull/3325